### PR TITLE
Add FAQ page

### LINKED ISSUE #46: FAQ update
We need to edit the FAQs to be more aligned to the mobile-first approach.

We also need to determine where that needs to live for launch (lockbox.firefox or txp).

### PR DESCRIPTION
**Pull request by devinreams**
Source: https://docs.google.com/document/d/1-h03Jq2Aq0ezMEqSVlfvDR8FTgnlThzy9UgXY8Fox9Q/edit?usp=sharing

Resolves #46 

screen shot 2018-05-22 at 12 23 03 pm

This creates a placeholder page at `/faq` with 11 questions.

It's also added to the beginning of the header / page navigation bar.

Each question can be linked-to using the ID for the question, such as `#question-11`.

### CODE REVIEW

**Comment by devinreams** [comment]
@mozilla-lockbox/product if it wasn't obvious to ya'll already — based on the FAQ titles provided — the FAQs URLs are now expected to be:

- https://lockbox.firefox.com/faq.html#what-is-lockbox
- https://lockbox.firefox.com/faq.html#how-do-i-use-lockbox
- https://lockbox.firefox.com/faq.html#how-do-i-enable-sync-on-firefox
- https://lockbox.firefox.com/faq.html#how-do-i-create-new-entries
- https://lockbox.firefox.com/faq.html#how-do-i-edit-existing-entries
- https://lockbox.firefox.com/faq.html#what-security-technology-does-lockbox-use

View Source then find the H2 `id`.

**Comment by devinreams** [comment]
@nickbrandt I just did a quick copy/paste job here to add the answers in. How does it look to you?

https://lockbox.firefox.com/faq.html

screen shot 2018-05-22 at 11 59 50 am

I wanted to check the page length and see if the link anchors consistently get the user "jumped" to the right spot.

**Comment by irrationalagent** [comment]
A little late to this party, but I believe logins are synced by default, meaning that once you sign up on desktop that option should already be checked. Would it then make sense to have language under "how do I enable sync on firefox" to be along the line of "Make sure that the logins item is checked" instead? Sorry if this conversation has already been had - feel free to ignore me then

File: faq.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: FAQ
+---
+
+## What is Lockbox?
+
+Lockbox is an experimental product from Mozilla, the makers of Firefox.  It’s a tool that allows you to access saved passwords from your mobile phone.  Right now, it’s available for iPhone and iPad, and soon, it will be available for Android.
+
```
[the remark this answers is not in the document]
Comment (devinreams) [reply faq.md]: >Recommendation from legal

Let's get this all in the originating Google Doc first and I'll just move one direction ➡️ into here.

**Comment by devinreams** [comment]
>I'll be discussing Leif's comment with Ryan and will get Michelle involved if we need to make some modifications to the enabling sync question/answer.

Cool, I'll hold off here until I hear from you all on a decision in the Google Doc itself (easier to edit there then just copy/paste over here).

>Also a good point about updating instances of "Lockbox" to "Firefox Lockbox"...from what I've heard about legal we are going to have to change this...which then changes a couple of our links 😞

Yeah, we'll just go back through and double check them all anyway, no biggie.

>Recommendation from legal: change "and soon.."

Updated in Google Doc and here (along with the renames).

---

@sandysage @nickbrandt Ready and waiting for your final decisions and approval to call one this d-u-n done!

**Comment by devinreams** [comment]
@sandysage @nickbrandt I don't see any updates in the Google Doc.

Is this OK to merge as-is so we can see it live on the website (via links in-app), make style changes, and do any other follow-up content updates in a separate PR?

**Pull request merged**